src/core/external/reveal/reveal-element.ts

```diff
@@ -107,7 +107,7 @@ class RevealElement extends SkyflowElement {
         if (data.name === this.#iframe.name) {
           callback({
             ...this.#metaData,
-            record: this.#recordData,
+            record: { ...this.#recordData, elementId: this.#elementId },
```

Comment:
instead of adding the elementID here. add it in line 72, as it will be stored along with the other recordData.